src/Pratt.elm

```diff
@@ -0,0 +1,445 @@
+module Pratt exposing
+    ( Config, expression
+    , subExpression
+    , literal, constant, prefix
+    , infixLeft, infixRight, postfix, recordAccessPostfix
+    )
+
+{-|
+
+  - **Expression parser**: [`expression`](#expression)
+  - **Configuration helpers**: [`subExpression`](#subExpression)
+      - **oneOf** helpers: [`literal`](#literal) [`constant`](#constant)
+        [`prefix`](#prefix)
+      - **andThenOneOf** helpers: [`infixLeft`](#infixLeft)
+        [`infixRight`](#infixRight) [`postfix`](#postfix)
+
+
+# Expression parser
+
+@docs Config, expression
+
+
+# Configuration helpers
+
+@docs subExpression
+
+
+## **oneOf** helpers
+
+@docs literal, constant, prefix
+
+
+## **andThenOneOf** helpers
+
+@docs infixLeft, infixRight, postfix, recordAccessPostfix
+
+-}
+
+import Combine exposing (Parser)
+import Elm.Parser.State as State exposing (State)
+import Elm.Syntax.Node exposing (Node)
+import Pratt.Advanced as Advanced
+
+
+
+-- EXPRESSION PARSER CONFIGURATION
+
+
+{-| An opaque type based on
+[`Pratt.Advanced.Config`](Pratt.Advanced#Config) holding the parser
+configuration.
+-}
+type alias Config state expr =
+    Advanced.Config state expr
```

Comment:
Since the main (only?) difference between `Pratt` and `Pratt.advanced` was the type, it doesn't really make sense to separate them anymore does it?
What would you think of merging the two? Do we keep the comments?
I know we will add the license information later on.

Reply:
Yes, we can merge them, good idea